base/mpfr.jl

```diff
@@ -588,56 +582,54 @@ for f in (:log, :log2, :log10)
                               "with a complex argument. Try ", $f, "(complex(x)).")))
         end
         z = BigFloat()
-        ccall(($(string(:mpfr_,f)), :libmpfr), Int32, (Ptr{BigFloat}, Ptr{BigFloat}, Int32), &z, &x, ROUNDING_MODE[])
+        ccall(($(string(:mpfr_,f)), :libmpfr), Int32, (Ref{BigFloat}, Ref{BigFloat}, Int32), z, x, ROUNDING_MODE[])
         return z
     end
 end
 
 function log1p(x::BigFloat)
     if x < -1
         throw(DomainError(x, string("log1p will only return a complex result if called ",
-                                    "with a complex argument. Try log1p(complex(x)).")))
+                          "with a complex argument. Try log1p(complex(x)).")))
```

Comment:
See https://github.com/JuliaLang/julia/pull/23288#discussion_r133716554 
in the file there are *2* different conventions on how this is broken up. I chose one of the two to make it consistent.